.github/workflows/sphinx-build-check.yml

```diff
@@ -1,18 +1,15 @@
 name: "Pull Request Docs Check"
 on:
   pull_request:
-  push:
-    branches:
-      - master
-  schedule:
-    # Run every morning to detect flakiness and broken dependencies
-    - cron: '23 5 * * *'
+
 
 jobs:
-  docs:
+  check-build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-    - uses: ammaraskar/sphinx-action@master
+    - uses: bmagyar/sphinx-pages@master
       with:
-        docs-folder: "."
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        create_readme: false
```

Comment:
Should this be automatically `false` when deploy=false  (nice-to-have)

Reply:
it is false by default, removed